builder/azure/arm/config.go

```diff
@@ -392,11 +393,33 @@ func assertRequiredParametersSet(c *Config, errs *packer.MultiError) {
 	/////////////////////////////////////////////
 	// Capture
 	if c.CaptureContainerName == "" {
-		errs = packer.MultiErrorAppend(errs, fmt.Errorf("An capture_container_name must be specified"))
+		errs = packer.MultiErrorAppend(errs, fmt.Errorf("A capture_container_name must be specified"))
+	}
+
+	captureContainerNameRegex := regexp.MustCompile("^[a-z0-9][a-z0-9\\-]{2,62}$")
+	if !captureContainerNameRegex.MatchString(c.CaptureContainerName) {
+		errs = packer.MultiErrorAppend(errs, fmt.Errorf("A capture_container_name must satisfy the regular expression '^[a-z0-9][a-z0-9\\-]{2,62}$'."))
```

Comment:
To DRY the code you can use `captureContainerNameRegex.String()` to get the source text. This way it stays in sync if the pattern is updated later.